### DISCUSSION TITLE
Support slashed-zero font-variant

### DISCUSSION THREAD

**Discussion by JackCA** [react-native-community/discussions-and-proposals#713] [Ideas]
It appears that the [`slashed-zero` font-variant property](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) is not currently supported. It would be nice it was :)

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. I think you should be posting this as a PR against the main [react native repo](https://github.com/facebook/react-native) that the code details of such an implementation could be discussed further.

That's the only way it can be discussed & reviewed with the FB team.
